mobx/lib/src/api/async/observable_stream.dart

```diff
@@ -332,7 +345,8 @@ class _ObservableStreamController<T> {
             name: '$name.status'),
         _valueType = Observable(_ValueType.value,
             context: context, name: '$name.valueType'),
-        _data = Observable(initialValue, context: context, name: '$name.data') {
+        _data = Observable<dynamic>(initialValue,
```

Comment:
why not `Observable<T?>`

Reply:
@pavanpodila 
https://github.com/mobxjs/mobx.dart/blob/cea7013fad1fcdfbb3213045db9be5e3f311f2f3/mobx/lib/src/api/async/observable_stream.dart#L452

Yes, I would like to use `Observable<T?>`, but `Object` type is assigned inside the `_onError` function.




```dart
  void _onError(Object error) {
    final actionInfo = _actions.startAction();
    try {
      _status.value = StreamStatus.active;
      _valueType.value = _ValueType.error;
      _data.value = error;
    } finally {
      _actions.endAction(actionInfo);
      _tryInsertInitialValue();
      _controller.addError(error);
    }
  }
```